pkg/system/syscall_windows.go

```diff
@@ -66,6 +67,22 @@ func IsWindowsClient() bool {
 	return osviex.ProductType == verNTWorkstation
 }
 
+// IsIoTCore returns true if the currently running image is based off of
+// Windows 10 IoT Core.
+// @engine maintainers - this function should not be removed or modified as it
+// is used to enforce licensing restrictions on Windows.
```

Comment:
This comment is copied from the above similar function. If we want to change the wording here, it should be done to both.